<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>63</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Looking for Scanner</string>
  </property>
  <property name="windowIcon">
   <iconset theme="scanner">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>0</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>looking for printer ...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
